generators/ci-cd/templates/.gitlab-ci.yml.ejs

```diff
@@ -134,11 +134,11 @@ gradle-package:
 #    stage: release
 #    variables:
 #        REGISTRY_URL: registry.gitlab.com
-#        IMAGE_TAG: $CI_REGISTRY_IMAGE:$CI_COMMIT_REF_SLUG:$CI_COMMIT_SHA
```

Comment:
Fun fact I was about to do something similar. Can't we use ` IMAGE_TAG: $CI_REGISTRY_IMAGE/$CI_COMMIT_REF_SLUG:$CI_COMMIT_SHA`? With that the gitlab container scanning should work out of box. 

If you are curious I have setup sample apps on gitlab.com where I would like to integrate the complete gitlab ci feature set like dependency scanning etc here

https://gitlab.com/atomfrede/jhipster-sample-app 

https://gitlab.com/atomfrede/jhipster-sample-app-gradle

Reply:
I think it is a question of favor. It would work, too. But then the actual image name would be 'master' or tag name etc. 

About the dep scanning, sure, but I'd like to have it kind of specialized for our applications...those templates are general and that leads to long CI times IMHO